### PR TITLE
add codegen for TinyGo actors

### PR DESCRIPTION
**Pull request by stevelr**
Signed-off-by: stevelr <steve@cosmonic.com>

Updates dependency on weld-codegen to 0.4.4 which is in PR https://github.com/wasmCloud/weld/pull/100

### CODE REVIEW

**Comment by autodidaddict** [comment]
Will review this when the relative path dependency is gone

**Comment by stevelr** [comment]
Build checks won't pass until  https://github.com/wasmCloud/weld/pull/100 is merged. Then, we can remove the relative path and re-trigger the build checks.

**Review by autodidaddict — DISMISSED** [review]

**Review by brooksmtownsend — DISMISSED** [review]
Good to go once weld-codegen is released

File: .rustfmt.toml [hunk 1]
```diff
@@ -1,2 +1,2 @@
-imports_granularity = "Crate"
+#imports_granularity = "Crate"
```
Comment (brooksmtownsend) [inline .rustfmt.toml]: Let's just remove this if we don't want it

**Review by brooksmtownsend — CHANGES_REQUESTED** [review]
only a little nit

File: .rustfmt.toml (hunk 1 above)
Comment (stevelr) [reply .rustfmt.toml]: Deleted. 
(I like it, but on rust stable it isn't supported and unrecognized options cause a hard fail)

**Review by brooksmtownsend — APPROVED** [review]

**Pull request merged**